scripts/create-cas-tgt.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+test -z "$2" && echo "Usage: $0 <username> <password> [<cas-server>]" && exit 1
+CAS_SERVER=virkailija.testiopintopolku.fi
+test -n "$3" && CAS_SERVER="$3"
+
+curl -s -D- -o /dev/null "https://$CAS_SERVER/cas/v1/tickets" \
+	-d "username=$1&password=$2" \
+| sed -ne 's/^[Ll]ocation: \([^ \r]*\).*$/\1/p' \
```

Comment:
Tää ei toimi BSD sed:n kanssa, pitää muuttaa

```suggestion
| sed -ne 's/^[Ll]ocation: \([^[:space:]]*\).*$/\1/p' \
```